cypress/e2e/features/backend/subscription.feature

```diff
@@ -2,11 +2,11 @@ Feature: Subscription Tiers API
 As a scrum team member, when I send a Subscription GET request, I should be able to retrieve all tiers information
 
   Scenario: Successful Subscription GET request
-    Given User should get refresh_token and create authToken
+    Given user should get refresh_token and create authToken
```

Comment:
If it is U (not u) in step definition, then this step will fail with `step definition is missing` error